pkg/start/start.go

```diff
@@ -315,15 +314,20 @@ func (o *Options) NewControllerContext(cb *ClientBuilder) *Context {
 	cvInformer := externalversions.NewFilteredSharedInformerFactory(client, resyncPeriod(o.ResyncInterval)(), "", func(opts *metav1.ListOptions) {
 		opts.FieldSelector = fmt.Sprintf("metadata.name=%s", o.Name)
 	})
-	cmInformer := informers.NewFilteredSharedInformerFactory(kubeClient, resyncPeriod(o.ResyncInterval)(), internal.ConfigNamespace, func(opts *metav1.ListOptions) {
+	cmConfigInformer := informers.NewFilteredSharedInformerFactory(kubeClient, resyncPeriod(o.ResyncInterval)(), internal.ConfigNamespace, func(opts *metav1.ListOptions) {
 		opts.FieldSelector = fmt.Sprintf("metadata.name=%s", internal.InstallerConfigMap)
 	})
+	cmManagedInformer := informers.NewFilteredSharedInformerFactory(kubeClient, resyncPeriod(o.ResyncInterval)(), internal.ConfigManagedNamespace, func(opts *metav1.ListOptions) {
+		opts.FieldSelector = fmt.Sprintf("metadata.name=%s", internal.UserCAConfigMap)
```

Comment:
I don't think we can force ourselves to only objects with name `UserCAConfigMap`.. I think we can create a list for maybe keep it for entire namespace to begin with.

Reply:
ack...will fix this tomorrow.